coordinator/src/main/scala/com/socrata/datacoordinator/service/DynamicPortMap.scala

```diff
@@ -1,15 +1,17 @@
 package com.socrata.datacoordinator.service
 
-import com.typesafe.scalalogging.slf4j.Logging
+import org.slf4j.Logger
 
-trait DynamicPortMap extends Logging  {
+trait DynamicPortMap {
+
+  val log: Logger
```

Comment:
This will still use the logger of the dynamic class (well, the class of whatever provides the implementation here)... at least it's more explicit about it, but I'd use a `private val log = getLogger(classOf[DynamicPortMap])` so that the log line comes out as originating in `csds.DynamicPortMap` instead of `csds.Main$`